platform/base/core/pom.xml

```diff
@@ -248,6 +242,10 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
```

Comment:
do we need to rely on org.jetbrains for this? isnt @NotNull an existing annotation in java?

Reply:
#671 handles that